db-persistence/my/SetupDatabase.ps1

```diff
@@ -30,6 +30,14 @@ if ($restartingInstance) {
                 if (($bakfile -ne "") -and $_.Name -eq 'CRONUS') {
                     return; # don't restore CRONUS if we have provided our own bak
                 }
+
+                # set recovery mode and shrink log
+                $sqlcmd = "ALTER DATABASE $($db.Name) SET RECOVERY SIMPLE WITH NO_WAIT"
+                & sqlcmd -Q $sqlcmd
+                $_.LogFiles | ForEach-Object {
+                    $sqlcmd = "DBCC SHRINKFILE (N'$($_.Name)' , 10) WITH NO_INFOMSGS"
```

Comment:
when I ran that manually I needed to prepend `USE $(db.Name) ;"` for that command to work

Reply:
shouldn't we shrink regularly? at least on every service restart but maybe also every day/week?

Reply:
I'll add the USE. With SIMPLE recovery mode, there is no need to shrink, because the log file should never grow

Reply:
@lippertmarkus PTAL again